MODX3 support (dont break in MODX2)

dimasites opened the pull request:
Replaced missing in MODX3 variable for its hardcoded value.

My test screenshots with comments:

![2022-03-30_22-43-57](https://user-images.githubusercontent.com/5102558/160921160-6a9e239c-9d1f-4c1e-82c3-4487d2402f4f.png)
but this variable missed in MODX3

![2022-03-30_22-13-17](https://user-images.githubusercontent.com/5102558/160921215-17e48e20-4466-4111-81f2-4a39e686077b.png)

![2022-03-30_22-27-14](https://user-images.githubusercontent.com/5102558/160921256-d499ad4f-b050-4ca7-a1d3-b5544248d467.png)

dimasites committed:
MODX3 support (dont break in MODX2)

Replaceв missing in MODX3 variable for its hardcoded value

legiero commented:
https://docs.modx.com/3.x/ru/extending-modx/menus/action-list reference to this changes

dimasites committed:
Update readme.md

Added fresh MODX-version screenshots

Commit:
Развернул оригинальные min-файлы

Перед изменениями

(cherry picked from commit 1028e3fa5229bdf1514f42bfaa6151e4dc438304)

Commit:
Добавлен поиск по modx-ресурсам в диалог link

Правки установщика пакета
Исправлена форма, JS-скрипты и CSS-стили (только для дефолтного скина monoo-lisa)
Добавлен процессор поиска

Commit:
Добавлена локализация поиска по modx-ресурсам

(cherry picked from commit a6116f84a828435ceb7c9b672c35511cdf6cdc00)

Commit:
Исправлена верстка TAB-ов диалога link

Уменьшена их ширина чтобы длинные названия не выталгивали вкладки за пределы окна

(cherry picked from commit 93b7e8f03dc579dd0a38e95cd3ce2a5b946b9087)

Commit:
Merge 'feature/link-modx-res-search'

dimasites committed:
Partial code update for 1.4.5 version

dimasites committed:
move files

dimasites committed:
Fix builder paths and minor refactoring

dimasites committed:
fix readme, this is install instuction only now

dimasites committed:
update path in builder

dimasites committed:
Add resolver

dimasites committed:
add resolver runner

dimasites committed:
Merge branches

dimasites committed:
Merge branches

dimasites committed:
Add missed code of prevous version

denius-dev committed:
Удалил лишние строки в билдере

dimasites committed:
Merge pull request #2 from denius-dev/develop

Удалил лишние строки в билдере

dimasites committed:
Update builder for better DX

Add submodule "modx-build-environment-gui"

dimasites committed:
update build env version

dimasites committed:
CKEditor 1.4.6 (2024.07.03)

====================================
- Fix small (as expected) editor frame height in some cases via default autogrow plugin settings
- Removed Flash plugin, because .swf format is outdated
- Added html5video plugin. Now we can paste .mp4 videos via editor!

dimasites committed:
Replace old style tags to actual

dimasites committed:
Replace дв style tags to actual

dimasites committed:
Обновленный appstat resolver под MODX3

dimasites committed:
Обновленный appstat resolver под MODX3

dimasites committed:
Update build env GUI

dimasites committed:
Update build env GUI

dimasites committed:
CKEditor 1.4.7 (draft)

==
- Old HTML elements replaced to actual by default in new (and only new) pasted or typed content.
- • <strong> replaced to <b> tag (thanks @dimasites)
- • <em> replaced to <i> tag
- Improved MODX3 support: fixed search by pagetitle in "paste link" dialog

dimasites committed:
CKEditor 1.4.7 (draft)

==
- Old HTML elements replaced to actual by default in new (and only new) pasted or typed content.
- • <strong> replaced to <b> tag (thanks @dimasites)
- • <em> replaced to <i> tag
- Improved MODX3 support: fixed search by pagetitle in "paste link" dialog

dimasites committed:
Merge branch 'master' of https://github.com/dimasites/modx-ckeditor into github-master

dimasites committed:
1.4.7-ce

Community Edition release

dimasites committed:
Added nbsp plugin

Now we can put non-breaking space by CTRL+Space keyboard input
